### PR TITLE
Refactor README and split installation and development

### PR DESCRIPTION
**Pull request by noirbizarre**
This PR actualize the README and distinguish installation and development

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,26 +16,62 @@ uData customizations for Etalab / Data.gouv.fr.
...
+├── udata
+│   ├── ...
+│   └── setup.py
+├── udata-gouvfr
+│   ├── ...
+│   └── setup.py
+└── udata.cfg
+```
+
+The next steps use the same Python virtual environment
```
Comment (davidbgk) [inline README.md]: The next step is to use

File: README.md [hunk 2]
```diff
@@ -14,26 +16,62 @@ uData customizations for Etalab / Data.gouv.fr.
...
+and the same version of npm (for JS) than `udata`.
```
Comment (davidbgk) [inline README.md]: as `udata`.

File: README.md [hunk 3]
```diff
@@ -14,26 +16,62 @@ uData customizations for Etalab / Data.gouv.fr.
...
+
+Clone the `udata-gouvfr` repository into your workspace
+and instal it in development mode:
```
Comment (davidbgk) [inline README.md]: install

File: README.md [hunk 4]
```diff
@@ -14,26 +16,62 @@ uData customizations for Etalab / Data.gouv.fr.
...
+```python
+PLUGINS = ['gouvfr']
+THEME = 'gouvfr'
+```
+
+## Development
+
+Prepare a [`udata` development environment][udata-develop].
+
+It is recommanded to have a workspace containing with the following layout:
```
Comment (davidbgk) [inline README.md]: recommended + workspace close to this layout:

File: README.md [hunk 5]
```diff
@@ -14,26 +16,62 @@ uData customizations for Etalab / Data.gouv.fr.
...
 
-Remain in the same virtual environment (for Python) and use the same version of npm (for JS).
+Remain in the same Python virtual environment
+and install **udata-gouvfr**:
+
+```shell
+pip install udata-gouvfr
+```
+
+Modify your local configuration file of **udata** (typically, `udata.cfg`) as following:
```
Comment (vinyll) [inline README.md]: _as follows:_ or _with the following:_

File: README.md [hunk 6]
```diff
@@ -14,26 +16,62 @@ uData customizations for Etalab / Data.gouv.fr.
...
+Clone the `udata-gouvfr` repository into your workspace
+and install it in development mode:
 
 ```shell
 git clone https://github.com/etalab/udata-gouvfr.git
 pip install -e udata-gouvfr
 ```
 
-Modify your local configuration file of **udata** (typically, `udata.cfg`) as following:
+Modify your local `udata.cfg` configuration file as following:
```
Comment (vinyll) [inline README.md]: same

**Pull request merged**